src/hooks/usePoolData.ts

```diff
@@ -334,3 +355,32 @@ export default function usePoolData(
 
   return poolData
 }
+
+async function getSgtApy(
```

Comment:
nit: this should be APR (also applies to references in the PR title/description)